app/stores/SchoolStore.js

```diff
@@ -9,6 +9,21 @@ import { EventEmitter } from 'events';
 const CHANGE_EVENT = 'change';
 let _schools = {};
 
+
+const schoolCheckDecorator = function(func, defaultValue) {
```

Comment:
I have changed the name in the next PR to getSchoolByIdWrapper... Do you come out with some other better name?